apps/ensindexer/src/lib/indexing-status/ponder-metadata/zod-schemas.ts

```diff
@@ -27,9 +27,9 @@ const makeChainNameSchema = (indexedChainNames: string[]) => z.enum(indexedChain
 
 const PonderBlockRefSchema = makeBlockRefSchema();
 
-const PonderCommandSchema = z.enum(["dev", "start"]);
+const PonderCommandSchema = z.enum(["dev", "start", "serve"]);
```

Comment:
added support for `ponder serve` because i don't want to always be indexing, it's very useful to be able to serve the already indexed schema